changelogs/fragments/zabbix_user-mediatype-error.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - zabbix_user - Now supported changed JSON structure of mediatype for since zabbix version 4.4 or more
```

I would rather go with something less specific here like this:
```suggestion
  - zabbix_user - Fixed an issue where module failed with zabbix 4.4 or above (see https://github.com/ansible/ansible/pull/67475)
```
What do you think?

---

I think it's a good suggestion!  
Thank you!